### DISCUSSION TITLE
API Returns no response and times out in production, not on local

### DISCUSSION THREAD

**Discussion by brogers1993** [weather-gov/api#684] [Q&A]
I've got a weird one, I'm having an issue where I can call the api.weather.gov endpoints from my local machine (and postman, etc.), but when I deploy my app to production (on fly.io), all requests time out/never return a response. I've been able to confirm I can successfully make requests to other domains via my production application (tested using [this resource](https://documenter.getpostman.com/view/8854915/Szf7znEe#068625c4-95f6-4e06-afb3-7ca055e72eb0) from postman), so this seems to be something specific to the weather.gov domain/API vs. a general access to external sites on my server. 

I'm also making 1 request each time I test, so I don't think this is a rate limiting. I am also using a hopefully fairly unique user-agent to identify my application. Are there any IP/port restrictions on the API or other suggested places I can look? My server is using an IPv6 address if that makes a difference.

**Comment by dwhitemv25** [comment]
See post #673, apparently Postman and Akamai don't get along.

**Reply by brogers1993** [reply]
Hmmm, I'm not sure if these are the same issue. #673 is at least getting a 403 response, I'm not getting ANYTHING back from the API.
